.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+sudo apt update && sudo apt install -y --no-install-recommends \
+    clang
```

Comment:
If moving this to a dockerfile, makes sure to also remove the cache to reduce the image size